src/main/java/frc/robot/subsystems/Manager/Manager.java

```diff
@@ -58,13 +79,19 @@ public void runState() {
         Logger.recordOutput("Manager/State", getState().getStateString());
         Logger.recordOutput("Manager/State Time", getStateTime());
 
+        if(Controllers.OPERATOR_CONTROLLER.getXButtonPressed()) {
```

Comment:
This should be a runnable trigger, right? Ideally no `if` statements in the `runState` function